doc/faq.rst

```diff
@@ -58,17 +58,32 @@ to 100. If there are ``maxPoolSize`` connections to a server and all are in
 use, the next request to that server will wait until one of the connections
 becomes available.
 
-The client instance opens one additional socket per server in your MongoDB
+The client instance opens two additional sockets per server in your MongoDB
```

Comment:
This change is because of the 4.4+ streaming SDAM behavior which requires 2 sockets per server.